pnpm-lock.yaml

```diff
@@ -68,10 +68,10 @@ importers:
         specifier: '=2.4.13'
         version: 2.4.13
       '@marinade.finance/validator-bonds-cli-core':
-        specifier: 'workspace: ^2.1.0'
+        specifier: 'workspace: ^2.1.1'
```

Comment:
All the packages are bumped to `2.1.2`, shouldn't that be reflected here in the lock too?

Reply:
Yes, thanks. updated